ops/web-e2e.cloudbuild.yaml

```diff
@@ -14,6 +14,19 @@
 
 
 steps:
+  - name: bash
```

Comment:
include `id` line to indicate what the step will be doing. Similar to https://github.com/GoogleCloudPlatform/emblem/blob/main/ops/terraform.cloudbuild.yaml 

example

id: copy client library files
... 
id: install python packages

etc

Reply:
Done.